src/api/Alert/index.ts

```diff
@@ -321,4 +323,16 @@ export class AlertAPI extends Request {
       params: params
     });
   }
+
+  /**
+   * Get all cases from alerts that a user has access to.
+   */
+  public getFullCaseList(this: Request<IFullCaseListResponse>, params: IGetFullCaseListParams) {
+    return this.request({
+      errors: [FEATURE_DISABLED, UNAUTHORIZED, DATA_SOURCE_UNAVAILABLE, FORBIDDEN, NOT_FOUND],
+      url: '/:sourceKey/alerts/cases/list',
+      method: 'GET',
+      params: params
```

Comment:
where is the special encoding for params.sortBy handled?

Reply:
@davidrapin 
Initially it was handled by the front-end  but now we figured that this might not be a good idea so we moved the encoding part to be handled by the res-client